Sources/Messenger/Response/Response.swift

```diff
@@ -0,0 +1,153 @@
+//
+//  Response.swift
+//  SwiftyBot
+//
+//  The MIT License (MIT)
+//
+//  Copyright (c) 2016 - 2019 Fabrizio Brancati.
+//
+//  Permission is hereby granted, free of charge, to any person obtaining a copy
+//  of this software and associated documentation files (the "Software"), to deal
+//  in the Software without restriction, including without limitation the rights
+//  to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+//  copies of the Software, and to permit persons to whom the Software is
+//  furnished to do so, subject to the following conditions:
+//
+//  The above copyright notice and this permission notice shall be included in all
+//  copies or substantial portions of the Software.
+//
+//  THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+//  IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+//  FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+//  AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+//  LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+//  OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+//  SOFTWARE.
+
+import BFKit
+import Foundation
+import Helper
+import Vapor
+
+/// Messenger response.
+public struct Response: Content {
+    /// Messaging type.
+    public private(set) var messagingType: MessagingType
+    /// Recipient.
+    public private(set) var recipient: Recipient?
+    /// Final response message.
+    public private(set) var message: MessageResponse
+    
+    /// Coding keys, used by Codable protocol.
+    private enum CodingKeys: String, CodingKey {
+        case messagingType = "messaging_type"
+        case recipient
+        case message
+    }
+    
+    public init() {
+        messagingType = .response
+        recipient = nil
+        message = .text("Unknown error.")
+    }
+}
+
+// MARK: - Response Extension
+
+/// Response extension.
+public extension Response {
+    /// Declared in an extension to not override default `init` function.
+    ///
+    /// - Parameter request: Message request.
+    /// - Throws: Decoding errors.
+    mutating func create(for request: Request) throws -> Future<Response> {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Function should have complexity 10 or less: currently complexity equals 12](https://app.codacy.com/app/FabrizioBrancati/SwiftyBot/pullRequest?prid=3796373)